src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -2,15 +2,22 @@
 
 import com.training.assignments.Assignment;
 import com.training.assignments.Operations;
+import com.training.assignments.inheritance.ParentChild;
+import com.training.assignments.polymorphism.overriding.Director;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class AssignmentTest extends BaseTest {
     private Logger logger = LoggerFactory.getLogger(this.getClass());
 
-   private Assignment assignment = new Assignment();
-   private Operations operation = new Operations();
+    private Assignment assignment = new Assignment();
+    private Operations operation = new Operations();
+    private ParentChild parentChild=new ParentChild();
+    private Director director;
```

Comment:
Define and instantiate both locally in test method since its used only for that method